src/tile/index.html

```diff
@@ -0,0 +1,12 @@
+<div class="gb-product-tile">
+  <a href="{ $product.link() }" onclick="{ $product.onClick }"><img riot-src="{ $product.data.image }" alt="" class="gb-product-tile__image"></a>
+  <div class="gb-product-tile__content">
+    <h3 class="gb-product-tile__content__title"><a href="{ $product.link() }" onclick="{ $product.onClick }">{ $product.data.title }</a></h3>
+    <span class="gb-product-tile__content__price">{ $product.data.price }</span>
```

Comment:
does this need to be `gb-product-tile__content__price`? why not `gb-product-tile__price`?